### ISSUE TITLE
L5 - Class tymon.validators.payload.factory does not exist

### ISSUE THREAD

**Issue by sebestenyb** [tymondesigns/jwt-auth#40]
Hello

I have the following problem with a pretty much from-scratch L5 project.
Any idea?

Cheers, Sebi

```	
public function test()
{
	$user = User::find(1);
	$token = JWTAuth::fromUser($user);
	return response()->json($token);
}
```

```
ReflectionException in Container.php line 776:
Class tymon.validators.payload.factory does not exist
in Container.php line 776
at ReflectionClass->__construct('tymon.validators.payload.factory') in Container.php line 776
at Container->build('tymon.validators.payload.factory', array()) in Container.php line 656
at Container->make('tymon.validators.payload.factory', array()) in Application.php line 572
at Application->make('tymon.validators.payload.factory') in Container.php line 1231
at Container->offsetGet('tymon.validators.payload.factory') in JWTAuthServiceProvider.php line 226
at JWTAuthServiceProvider->Tymon\JWTAuth\Providers\{closure}(object(Application)) in Container.php line 290
at Container->Illuminate\Container\{closure}(object(Application), array()) in Container.php line 773
at Container->build(object(Closure), array()) in Container.php line 656
at Container->make('tymon.jwt.payload.factory', array()) in Application.php line 572
at Application->make('tymon.jwt.payload.factory') in Container.php line 1231
at Container->offsetGet('tymon.jwt.payload.factory') in JWTAuthServiceProvider.php line 175
at JWTAuthServiceProvider->Tymon\JWTAuth\Providers\{closure}(object(Application)) in Container.php line 290
at Container->Illuminate\Container\{closure}(object(Application), array()) in Container.php line 773
at Container->build(object(Closure), array()) in Container.php line 656
at Container->make('tymon.jwt.manager', array()) in Application.php line 572
at Application->make('tymon.jwt.manager') in Container.php line 1231
at Container->offsetGet('tymon.jwt.manager') in JWTAuthServiceProvider.php line 190
at JWTAuthServiceProvider->Tymon\JWTAuth\Providers\{closure}(object(Application)) in Container.php line 290
at Container->Illuminate\Container\{closure}(object(Application), array()) in Container.php line 773
```

**Comment by tymondesigns** [comment]
Ahh my bad.. I think I broke this last night.

This should now be fixed as of - b7d4731e3b4f6047d7ba999501df53857348e4eb

just run composer update until the new code is pulled in

Thanks

**Comment by sebestenyb** [comment]
Ah, that's what's called good service :+1: 
Fixed the issue, thanks!
